src/containers/ArticlePage/components/ContentField.tsx

```diff
@@ -99,7 +99,9 @@ const ContentField = ({ field, form }: Props) => {
     <>
       <FieldHeader title={t('form.relatedContent.articlesTitle')} />
       <ElementList
-        elements={relatedContent}
+        elements={relatedContent.filter(
+          (rc): rc is DraftApiType | RelatedContentLink => typeof rc !== 'number',
```

Comment:
Jeg er litt redd for at å ikke være eksplisitt med typen på typeguards som disse fort kan gjøre at vi ender opp med å caste andre objekter til en type uten at det egentlig stemmer. Feks hvis `relatedContent` fikk enda en type i unionen sin.
Hva tenker du?
```suggestion
          (rc: number | DraftApiType | RelatedContentLink): rc is DraftApiType | RelatedContentLink => typeof rc !== 'number',
```